examples/rust/bzsstack/src/bpf/profile.bpf.c

```diff
@@ -0,0 +1,40 @@
+// SPDX-License-Identifier: GPL-2.0 OR BSD-3-Clause
```

Comment:
let's just symlink this to profile.bpf.c from C example, why maintaining two copies?